tests/e2e/plugins/tracking-opt-in.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Plugin Name: E2E Tests User Tracking Opt-in
+ * Plugin URI:  https://github.com/google/site-kit-wp
+ * Description: Utility plugin for faking user tracking opt-in during E2E tests.
+ * Author:      Google
+ * Author URI:  https://opensource.google.com
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+add_filter(
+	'get_user_option_googlesitekit_tracking_optin',
+	'__return_true'
+);
```

Comment:
Why is this needed? I thought user surveys are being shown regardless, we only show a more explicit consent note on them when the user is _not_ opted in.

Reply:
Hmm, you're right. I misread this guard before when I was trying to get it to work.
https://github.com/google/site-kit-wp/blob/202d55f0a1f19f405a82fdfb8576e0ab02464080/assets/js/components/surveys/CurrentSurvey.js#L230-L237

I'll remove it as it's not needed 👍